src/benchmarks/real-world/Microsoft.ML.Benchmarks/KMeansAndLogisticRegressionbench.cs

```diff
@@ -16,6 +17,7 @@ namespace Microsoft.ML.Benchmarks
     /// training a Logistic Regression model on the derived features plus the score from the
     /// KMeans trainer.
     /// </summary>
+    [BenchmarkCategory(Categories.MachineLearning)]
```

Comment:
same as above: we don't need categories if there is only one category in given project

Reply:
See above.